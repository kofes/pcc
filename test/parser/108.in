function foo(var i,j : real; k : real): boolean;
begin
end;

function foo(i,j,k : real):integer;
begin
end;

begin
end
