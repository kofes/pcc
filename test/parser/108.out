Duplication of function "FOO" in pos (5, 10);
